src/test/java/com/zemnov/salon/SalonApplicationTests.java

```diff
@@ -1,13 +1,49 @@
 package com.zemnov.salon;
 
+import com.zemnov.salon.model.Master;
+import com.zemnov.salon.repository.MasterRepo;
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 
+import java.util.Optional;
+
 @SpringBootTest
 class SalonApplicationTests {
```

Comment:
Seems like here presented only 2 test and all those about masterRepository. So let's name it 'MasterRepositoryIntTest'. But it will be better if we will create such test for service, not for an repository, becasue repository doesn't do any logic currently written by you